ide/css.editor/src/org/netbeans/modules/css/editor/module/main/BrowserSupportModule.java

```diff
@@ -66,7 +66,12 @@ public Collection<String> getPropertyNames(FileObject file) {
 
     @Override
     public PropertyDefinition getPropertyDefinition(String propertyName) {
-        return parser.getVendorSpecificProperties().get(propertyName);
+        PropertyDefinition pd = parser.getVendorSpecificProperties().get(propertyName);
```

Comment:
what is the result of this execution if propertyName is null? can it ever be null?

Reply:
I don't understand what is the relevance of this question. If there was a problem before this change it is still there. This is not the point of the changeset.

To answer the question: `BrowserSpecificDefinitionParser#getVendorSpecificProperties` is backed by a `HashSet`. This allows `null` as key. The parser though will not yield `null` keys, so the `null` will not have a backing value, yielding `null` fro the `PropertyDefinition`. This in turn is caught in the new line 70, which explicitly checks for `null`.

Reply:
ok - good 
 just saw a lack of null check and though there might be some potential NullPointer there.